src/contracts/index.ts

```diff
@@ -9,7 +9,10 @@ function getStableConverterContract(): StablecoinConverter {
   const stableCoinContractAddress = STABLE_COIN_CONTRACT_ADDRESS as string
 
   const abi = require('./StablecoinConverter.json')
-  return new web3.eth.Contract(abi, stableCoinContractAddress) as StablecoinConverter
+
+  // FIXME: There's an issue with this conversion
```

Comment:
No idea why it requires `unknown`. Not necessary in dex-react

Reply:
I showed to nick, also knowledgeable in TS and struggled to understand why

Thanks for trying anyways